src/app/features/languageSlice.ts

```diff
@@ -6,10 +6,12 @@ interface ILanguageState {
     currentLanguage: LanguageCode;
 }
 
+const localStorageLanguage = localStorage.getItem('language');
 const initialState: ILanguageState = {
-    currentLanguage: i18n.language as LanguageCode.English,
+    currentLanguage: localStorageLanguage
+        ? (localStorageLanguage as LanguageCode)
+        : (i18n.language as LanguageCode.English),
```

Comment:
casting needed? we could use type gurard

Reply:
Okay, I have removed the type assertions